### ISSUE TITLE
 Parameter setting problem

### ISSUE THREAD

**Issue by little1TouwaErio** [ChengBinJin/V-GAN-tensorflow#18]
In the utils.py file, there is a piece of code around 130 lines like this ： 
mean = np.mean(fundus_imgs[idx, ...][fundus_imgs[idx, ..., 0] > 40.0], axis =0)
Std = np.std(fundus_imgs[idx, ...][fundus_imgs[idx, ..., 0] > 40.0], axis=0)
How is this 40 selected? Can you please explain in detail?
Is there any special need to pay attention to the selection of this value?
I look forward to your reply as soon as possible. I will be very grateful for your help. @ChengBinJin

**Comment by ChengBinJin** [comment]
@little1TouwaErio I just followed the author's implementation in keras. 
Usually, this kind of threshold is decided by experimental observations.
